docs/source/workflow_outputs.rst

```diff
@@ -202,14 +202,14 @@ Based on which calculations were requested in the HPXML file, CSV annual output
   ENERGY STAR       TargetHome/results/ReferenceHome.csv                 ERI Reference Home for the ENERGY STAR Reference Design.
   ENERGY STAR       TargetHome/results/IndexAdjustmentHome.csv           ERI Index Adjustment Design for the ENERGY STAR Reference Design.
   ENERGY STAR       TargetHome/results/IndexAdjustmentReferenceHome.csv  ERI Index Adjustment Reference Home for the ENERGY STAR Reference Design.
-  ZERH              RatedHome/results/RatedHome.csv                      ERI Rated Home for the ZERH rated home.
-  ZERH              RatedHome/results/ReferenceHome.csv                  ERI Reference Home for the ZERH rated home.
-  ZERH              RatedHome/results/IndexAdjustmentHome.csv            ERI Index Adjustment Design for the ZERH rated home.
-  ZERH              RatedHome/results/IndexAdjustmentReferenceHome.csv   ERI Index Adjustment Reference Home for the ZERH rated home.
-  ZERH              TargetHome/results/RatedHome.csv                     ERI Rated Home for the ZERH Target Home.
-  ZERH              TargetHome/results/ReferenceHome.csv                 ERI Reference Home for the ZERH Target Home.
-  ZERH              TargetHome/results/IndexAdjustmentHome.csv           ERI Index Adjustment Design for the ZERH Target Home.
-  ZERH              TargetHome/results/IndexAdjustmentReferenceHome.csv  ERI Index Adjustment Reference Home for the ZERH Target Home.
+  DENH              RatedHome/results/RatedHome.csv                      ERI Rated Home for the DENH (formerly ZERH) rated home.
+  DENH              RatedHome/results/ReferenceHome.csv                  ERI Reference Home for the DENH (formerly ZERH) rated home.
```

Comment:
It's kind of excessive to mention that DENH was formerly ZERH in _every single reference_. Just mentioning it once on each page should be enough. I'm going to back out some of these.